changelog/add-8870-ece-for-shortcode-cart-and-checkout

```diff
@@ -1,4 +0,0 @@
-Significance: minor
-Type: add
-
-Add support for Express Checkout Element on shortcode Cart and Checkout pages.
```

Comment:
Can you expand on why we're removing this change log?

Reply:
This file was added by mistake in https://github.com/Automattic/woocommerce-payments/pull/8937 because the PR was based on the same branch @reykjalin used for his own [PR](https://github.com/Automattic/woocommerce-payments/pull/8914). 

Context: p1718967918819489-slack-C01BZUL57SQ

Reply:
Nevermind, it was removed here https://github.com/Automattic/woocommerce-payments/pull/9054